gravitee-am-model/src/main/java/io/gravitee/am/model/EnrollmentSettings.java

```diff
@@ -27,15 +27,29 @@
 @Setter
 @NoArgsConstructor
```

Comment:
we can remove annotation or setters / getters

Reply:
setters/getters removed